app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,53 @@
+class PagesController < ApplicationController
+    before_action :set_page, only: [:show, :edit, :update, :destroy]
+
+    def index
+        @pages = Page.all
+    end
+
+    def show
+        #set_page
+        # @page = Page.find(params[:id]) 
+        #render text: @page.title
+    end
+
+    def new 
+        @page = Page.new
+    end
+
+    def create
+        @page = Page.new(page_params)
+        @page.save
```

Comment:
I might consider using `save!` here if you dont want to handle a failure case in a custom way.  The way its written currently if the save fails the user would have no idea why

Reply:
or alternatively you could handle it in a similar way to how you do in the `PostsController`